src/commands/git-diff-set-parameters.yml

```diff
@@ -5,10 +5,14 @@ parameters:
   base-revision:
     type: string
     description: Compare this branch to its ancestor named
-  output-path:
-    type: string
   mapping:
     type: string
+  config-path:
+    type: string
+  all-mappings-match-if-config-path-changed:
+    type: enum
+    default: ""
+    enum: ["", "true"]
```

Comment:
@caugustus I switched from boolean to enum (a pattern already used in this orb) because the documentation about what a boolean will actually render as is hard to find (and in practice it surprisingly goes to 1/0).